packages/patternfly-4/react-core/src/components/TextArea/TextArea.tsx

```diff
@@ -15,8 +16,13 @@ export interface TextAreaProps extends Omit<HTMLProps<HTMLTextAreaElement>, 'onC
   className?: string;
   /** Flag to show if the TextArea is required. */
   isRequired?: boolean;
-  /** Flag to show if the TextArea is valid or invalid. */
+  /** Flag to show if the TextArea is valid or invalid. This prop will be deprecated you should use validated instead. */
   isValid?: boolean;
+  /** Value to indicate if the input is modified to show that validation state.
+   * If set to success, input will be modified to indicate valid state. If set to success, input will be modified to indicate valid state.
+   * If set to error,  text color of helper text will be modified to indicate error state.
+   */
+  validated?: 'success' | 'error' | 'default' | ValidatedOptions;
```

Comment:
What is `ValidatedOptions`? It isn't in the docs for TextInput or Select.

Reply:
Nice catch!